src/lib/Behat/PageElement/Dialog.php

```diff
@@ -14,12 +14,15 @@ class Dialog extends Element
     /** @var string Name by which Element is recognised */
     public const ELEMENT_NAME = 'Dialog';
 
-    public function __construct(UtilityContext $context)
+    public $dialogContainer;
+
+    public function __construct(UtilityContext $context, ?string $containerLocator = 'body')
```

Comment:
I'll check it for Dialog and in this case you are probably right, but for Admin list it's not the only reason - there are pages that has two admin lists (not in tabs like here), like content type details or section details, and in that case we need to specify on which we want to perform actions. 
Actually, it was working till now for ContentTypes because we had generic locators, and we checking specific records from tables, but in fact, in most cases we searched through both lists instead of the one that we specified.